packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -21,6 +21,10 @@ enum CollapseMode {
 
   /// The background widget will act as normal with no collapsing effect.
   none,
+
+  /// The background widget will first be collapsed with the [CollapseMode.none]
+  /// effect and then replaced with the [CollapseMode.pin] effect
+  nonePin,
```

Comment:
I am not sure I understand what the expected behavior is based on this name.

Reply:
I think it is not a good choice to use [none] directly, everyone is used to the characteristics of [none]. So what should this be used instead? [noneToPin]? I'm very bad at naming.